contracts/operators/Beefy/BeefyVaultOperator.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "./../../libraries/ExchangeHelpers.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title Beefy Single Vault Operator
+/// @notice Deposit/Withdraw in a Beefy vault (native or non-native).
```

Comment:
The main argument is to avoid the operator storage, with `token` and `vault` `immutable` (save gas), and also having the same process for adding/removing operations.
But it's easier to handle one operator with multiple vaults. So I think that both are fine.

Reply:
While operating the protocol, it is probably easier to manage one operator + one storage contract, rather than 10s of operators. This is true for the admin deploying contract but it also makes it easier to plug new vaults to the frontend, and route the users between the vaults

I understand it will be slightly more costly for users though. But relatively to what Beefy requires as gas, I'm not sure it would make a significant difference.

Reply:
ok let's do this!